datadog/resource_datadog_logs_custom_pipeline.go

```diff
@@ -1205,8 +1218,10 @@ func buildDatadogFilter(tfFilter map[string]interface{}) *datadogV1.LogsFilter {
 
 func getPipelineSchema(isNested bool) map[string]*schema.Schema {
 	return map[string]*schema.Schema{
-		"name":       {Type: schema.TypeString, Required: true},
-		"is_enabled": {Type: schema.TypeBool, Optional: true},
+		"name":        {Type: schema.TypeString, Required: true},
+		"is_enabled":  {Type: schema.TypeBool, Optional: true},
+		"tags":        {Type: schema.TypeList, Optional: true, Elem: &schema.Schema{Type: schema.TypeString}},
```

Comment:
```suggestion
		"tags":        {Type: schema.TypeSet, Optional: true, Elem: &schema.Schema{Type: schema.TypeString}},
```

Reply:
@skarimo if we use a set isn't it a risk of changing the order of the tags?
The set has the advantage of unicity but the pipelines API already does dedup if necessary